client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -29,75 +58,149 @@ const EditProfile = (props: Props): JSX.Element => {
           <Typography variant="h3" className={classes.heading}>
             Edit Profile
           </Typography>
-          <form>
-            <Grid container>
-              <Grid item xs={2} md={2}></Grid>
-              <Grid item xs={2} md={2} className={classes.formLabel}>
-                FIRST NAME
-              </Grid>
-              <Grid item xs={6} md={6} className={classes.formInputField}>
-                <TextField size="small" variant="outlined" placeholder="First Name" fullWidth></TextField>
-              </Grid>
-              <Grid xs={2} md={2}></Grid>
+          <Formik
+            initialValues={{
+              firstName: props.firstName,
+              lastName: props.lastName,
+              email: props.lastName,
+              phone: props.phone,
+              city: props.city,
+            }}
+            validationSchema={Yup.object().shape({
+              firstName: Yup.string().required('First name is required').max(20, 'First name is too long'),
+              lastName: Yup.string().required('Last name is required').max(20, 'Last name is too long'),
+              email: Yup.string().required('Email is required').email('Email is not valid'),
+              phone: Yup.number().notRequired(),
+              city: Yup.string().notRequired(),
+            })}
+            onSubmit={handleSubmit}
+          >
+            {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+              <form onSubmit={handleSubmit} noValidate>
+                <Grid container>
+                  <Grid item xs={2} md={2}></Grid>
+                  <Grid item xs={2} md={2} className={classes.formLabel}>
+                    FIRST NAME
+                  </Grid>
+                  <Grid item xs={6} md={6} className={classes.formInputField}>
+                    <TextField
```

Comment:
This is a suggestion you can implement in the future, you don't have to make changes in this PR.
Any components which we reuse multiple times can be modularized, in this case it would be the `TextField`, you can modularize this by creating another component. For eg: `CustomTextField`